src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -214,8 +248,35 @@ const CompleteIncident = props => {
               name="src"
               value={formValues.src || ' '}
             />
+            <br />
           </>
         )}
+
+        {/* Tags */}
+        {!editing ? (
+          <div className="dropdown-text-wrap">
+            <p className="complete-incident-dropdown-titles-bold">Tags</p>
+            <div>
+              {incident.tags ? incident.tags.join(", ") : ''}
```

Comment:
If incident.tags is empty, won't incident.tags.join(', ') evaluate to '' anyways?

Reply:
Yeah, but not if tags was null or undefined (for some reason)

Reply:
Yeah I suppose null doesn't have a join method. Was that a problem you ran into?

Reply:
In CompleteIncident.test.js, the incident data they are using doesn't have a tags property, and neither does any of the data in mockStoreData. That's probably something I should fix, huh?